datafusion/proto/tests/cases/roundtrip_physical_plan.rs

```diff
@@ -281,17 +283,6 @@ fn roundtrip_window() -> Result<()> {
         Arc::new(window_frame),
     ));
 
-    let plain_aggr_window_expr = Arc::new(PlainAggregateWindowExpr::new(
```

Comment:
I think this should not be removed 🤔

Reply:
I have added a test case in `fn roundtrip_window` in `datafusion/proto/tests/cases/roundtrip_logical_plan.rs`.

Reply:
@dharanad. The error is because that you didn't handle UDAF, so it falls back to builtin which results in NotImplementedError. We should handle UDAF for `PlainAggregateWindowExpr`, you can see how `SlidingAggregateWindowExpr` handle it

https://github.com/apache/datafusion/blob/08e4e6ad020136cac18710d0a18f31d44e025457/datafusion/proto/src/physical_plan/to_proto.rs#L165-L194

Reply:
Thanks for guiding me over here, i made some bad assumptions. I will handle this case